ShopifySharp/ShopifySharp.csproj

```diff
@@ -1,15 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+		<TargetFrameworks>netstandard2.0;net6.0;net7.0</TargetFrameworks>
 		<AssemblyName>ShopifySharp</AssemblyName>
 		<RootNamespace>ShopifySharp</RootNamespace>
 		<VersionPrefix>6.6.1</VersionPrefix>
 		<PublishRepositoryUrl>true</PublishRepositoryUrl>
 		<IncludeSymbols>true</IncludeSymbols>
 		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
-	</PropertyGroup>
-	<PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
-		<LangVersion>9.0</LangVersion>
+        <LangVersion>Latest</LangVersion>
```

Comment:
I didn't mean to merge this in, but I actually want to confirm this: I'm able to build the netstandard2.0 project using nullable references and `<LangVersion>Latest</LangVersion>`. Will these nullable references work in .NET Framework? Even if they don't, if the package itself still continues to function the same in DNF but gives everything else nullable references, that would be a huge improvement.

Reply:
I'm going to risk it and merge this in, since I have no way to test DNF on a Mac. We'll let the test suite run and see what happens.